File: 프로그래머스/lv1/12901. 2016년/2016년.java
class Solution {
    public String solution(int a, int b) {
        String[] answer = {"FRI","SAT","SUN","MON","TUE","WED","THU"};
        int[] months = {31,29,31,30,31,30,31,31,30,31,30,31};
        int days = 0;
        for(int i=1;i<a;i++){
            days += months[i-1];
        }
        days += b-1;
        
        return answer[days%7];
    }
}

File: JavaScript/프로그래머스/1/12901. 2016년/2016년.js
function solution(a, b) {
    const week = ['SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT'];
    const day = '2016-' + a + '-' + b;
    
    return week[new Date(day).getDay()];
}